### PR TITLE
Convert all source/**/*.html to .md

### PR DESCRIPTION
**Pull request by levitte**
Partial fix of #324

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+<table>
+  <tr>
+    <td>KBytes&nbsp;</td>
+    <td>Date&nbsp;&nbsp;</td>
+    <td>File&nbsp;</td>
+  </tr>
+  <!--#include virtual="index.inc" -->
+</table>
+
```
Comment (t8m) [inline source/index.md:49]: Somehow the table is merged with the next paragraph in the https://beta.openssl.org/source/ which looks weird.

File: source/gitrepo.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+checkout commands. See the discussion of branch naming below for more
+information.
+
+On Windows, once the repository is cloned, you should ensure that line
+endings are set correctly:
+
+    $ cd openssl
+    $ git config core.autocrlf false
+    $ git config core.eol lf
+    $ git checkout .
```
Comment (t8m) [inline source/gitrepo.md]: This (and the shell code above) does not seem to be properly rendered on https://beta.openssl.org/source/gitrepo.html

File: source/license.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: License
+---
+# License
```
Comment (t8m) [inline source/license.md:4]: Extra # here

Comment (t8m) [reply source/license.md:4]: Also on beta.openssl.org the title is duplicated.

File: source/mirror.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+| Locale | URL                                                |
+|--------|----------------------------------------------------|
+| CZ     | <ftp://ftp.fi.muni.cz/pub/openssl/>                |
+| DE     | <ftp://ftp.pca.dfn.de/pub/tools/net/openssl/>      |
+| DE     | <http://artfiles.org/openssl.org/>                 |
+| HR     | <ftp://ftp.linux.hr/pub/openssl/>                  |
+| HU     | <ftp://ftp.kfki.hu/pub/packages/security/openssl/> |
+| PL     | <ftp://guest.kuria.katowice.pl/pub/openssl/>       |
+| PL     | <rsync://ftp.tpnet.pl/pub/security/openssl/>       |
+
```
Comment (t8m) [inline source/mirror.md:20]: For some reason on beta.openssl.org the table is merged with the paragraph after it. Perhaps the md->html conversion should handle this?

Comment (t8m) [reply source/mirror.md:20]: Also the table styling has some glitches.

**Review by t8m — CHANGES_REQUESTED** [review]

File: source/gitrepo.md (hunk 2 above)
Comment (levitte) [reply source/gitrepo.md]: Okie

File: source/license.md (hunk 3 above)
Comment (levitte) [reply source/license.md:4]: Ah, it was a bad idea to use `title:`, as this is understood by pandoc.  I've just submitted #350, which uses `breadcrumb` instead, which is actually what it's used for...  I'll have to adjust all the pages now...

File: source/mirror.md (hunk 4 above)
Comment (levitte) [reply source/mirror.md:20]: There's a lot of weirdness happening with tables...  it's not in the generation, that looks perfectly fine, so it's something else.  In the HTML, this was solved with an added `<p>&nbsp;</p>` after the table, which is indeed an ugly workaround...

Re the rest of the glitches, I'll adjust that tomorrow

File: source/license.md (hunk 3 above)
Comment (levitte) [reply source/license.md:4]: I hope I've fixed it, together with #350.

File: source/mirror.md (hunk 4 above)
Comment (levitte) [reply source/mirror.md:20]: It seems that surrounding any table with \<p>...\</p> fixes the problem

File: source/license.md (hunk 3 above)
Comment (t8m) [reply source/license.md:4]: There is still the extra `#`

File: source/mirror.md (hunk 4 above)
Comment (t8m) [reply source/mirror.md:20]: It was resolved for the downloads table but not for the table with the list of the mirrors. Please look at https://beta.openssl.org/source/mirror.html

File: source/license.md (hunk 3 above)
Comment (t8m) [reply source/license.md:4]: Or you need to drop the `--------` under `# License`

Comment (levitte) [reply source/license.md:4]: Oh!  Yeah

Comment (levitte) [reply source/license.md:4]: Done

File: source/mirror.md (hunk 4 above)
Comment (levitte) [reply source/mirror.md:20]: Fixed

**Comment by levitte** [comment]
I discovered that the inclusion of legalities.shtml was inadvertently dropped. Fixed

**Comment by levitte** [comment]
All major glitches gone.  Final review, @openssl/omc?

**Comment by levitte** [comment]
Ping

**Comment by levitte** [comment]
Rebasee

**Comment by levitte** [comment]
Rebased and squashed

**Comment by levitte** [comment]
Ping

**Review by mattcaswell — APPROVED** [review]

**Comment by levitte** [comment]
Merged

239296da4d2d17929d7b5285b749cdeef3e44ad6 Convert all source/*.html to .md
fe6c01f3d893a28ccadd090f3908fe11b7dd6d3f Convert all source/old/*.html to .md
a18e42645868ebce68957fc86cdcafe613016931 $(SRCLISTS) isn't phony!

**Pull request closed without merging**